source/whats-new.txt

```diff
@@ -28,6 +29,25 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+
+.. _version-2.25:
+
+What's New in 2.25
+------------------
+
+The 2.25 driver release includes the following new features:
+
+- Added support for the ``MONGODB-OIDC`` authentication mechanism and automatic
+  token acquisition for Azure Identity Provider.
+- Added the class name to the error message reported when
+  ``BsonClassMapSerializer`` cannot locate a matching creator.
+- Added a ``LoggedStages`` field to surface the MQL executed after performing
+  a LINQ query.
+- Added support for overriding the ``"mongodb"`` service name with a custom one
+  by using the ``srvServiceName`` connection option.
+- Fixed behavior in which bulk write operations enumerated the requests
+  parameter more than once.
```

Comment:
I think 'Improved' instead of 'fixed' is better, as it technically it was not wrong.